core/templates/dev/head/pages/exploration_editor/settings_tab/settings_tab.html

```diff
@@ -286,11 +286,12 @@ <h3>Roles</h3>
                   </span>
                 </div>
 
-                <input type="submit" class="btn btn-default protractor-test-save-role" value="Save">
+                <input type="submit" class="btn btn-default protractor-test-save-role" value="Save" ng-disabled="!isTitlePresent()">
                 <button type="button" class="btn btn-default" ng-click="closeEditRolesForm()"
                         ng-show="isRolesFormOpen">
                   Cancel
                 </button>
+                <p ng-show="!isTitlePresent()" class="text-danger">Please provide a title before inviting.</p>
```

Comment:
are we going to show this message all the time in the settings tab (until the title is set)?
If no, ignore the next sentences. 
If yes, I think there are lots of reasons for which users come to the settings tab I'm not sure whether it's good to show this warning all the time. We should show this once they want to add a role to exploration (I see a similar pattern for the google doc share functionality).

@nithusha21, what do you think?

Reply:
@DubeySandeep , No we will not show this message all the time 
 I think this will show up only when a user clicks on "edit roles button" which is only used to add a user as a member.
![Screenshot from 2019-03-17 09-35-34](https://user-images.githubusercontent.com/43315144/54485065-69df9500-4898-11e9-8973-8e05f5059d74.png)
this opens on clicking ||
![Screenshot from 2019-03-17 09-36-44](https://user-images.githubusercontent.com/43315144/54485066-6b10c200-4898-11e9-8ad3-78af14044eeb.png)

Thanks!

Reply:
Ah, I see thanks for the explanation and attaching the screenshot.